src/System.Private.ServiceModel/src/Internals/System/Runtime/ActionItem.cs

```diff
@@ -75,35 +130,44 @@ public static Action<object> InvokeCallbackAction
                 }
             }
 
-            [Fx.Tag.SecurityNote(Critical = "Called by the scheduler without any user context on the stack")]
             private static void InvokeCallback(object state)
```

Comment:
Why remove SecurityNoteCritical?

Reply:
This is a hangover from the .NET Framework source code where there were analyzers and rules to verify correctness for partial trust. This was an annotation to tell the static analyzer that this method is safe despite what it thinks and the text was for the justification. Partial trust is no more, so all these attributes are irrelevant now.